src/spikeinterface/sorters/external/mountainsort5.py

```diff
@@ -201,6 +206,11 @@ def _run_from_folder(cls, sorter_output_folder, params, verbose):
                 sorting = ms5.sorting_scheme3(recording=recording_cached, sorting_parameters=scheme3_sorting_parameters)
             else:
                 raise ValueError(f"Invalid scheme: {scheme} given. scheme must be one of '1', '2' or '3'")
+        except Exception as e:
```

Comment:
The issue is for Dandi you had an OSError, for Windows we get PermissionError.

Reply:
@zm711 since we're not handling them differently here (just printing the error message), I don't think there's the need to differentiate. Unless you wanna handle any specific error right here?

Reply:
I mean the issue is that if someone triggers the ValueError it will trigger the except when it shouldn't. If someone puts in the wrong scheme we need to actually error out rather than just move onto the except @luiztauffer

Reply:
@zm711 right, I moved the scheme check outside ot the try/except block now